editor/src/components/filebrowser/filebrowser.tsx

```diff
@@ -101,26 +101,20 @@ function collectFileBrowserItems(
       ) {
         const codeResult = codeResultCache.cache[fullPath]
         if (codeResult != null) {
-          if (codeResult.error == null) {
-            Object.keys(codeResult.exports).forEach((exportedVar) => {
-              if (exportedVar != 'default') {
-                const exportedVarValue = codeResult.exports[exportedVar].value
-                const exportedVarIsFunction = typeof exportedVarValue === 'function'
-                fileBrowserItems.push({
-                  path: `${fullPath}/${exportedVar}`,
-                  type: 'export',
-                  fileType: null,
-                  typeInformation: codeResult.exports[exportedVar].type,
-                  originatingPath: originatingPath,
-                  hasErrorMessages: false,
-                  modified: false,
-                  exportedFunction: exportedVarIsFunction,
-                })
-              }
-            })
-          } else {
-            console.warn(`Error parsing JS file "${fullPath}": ${codeResult.error.toString()}`)
-          }
+          Object.keys(codeResult.exports).forEach((exportedVar) => {
+            if (exportedVar != 'default') {
+              fileBrowserItems.push({
+                path: `${fullPath}/${exportedVar}`,
+                type: 'export',
+                fileType: null,
+                typeInformation: codeResult.exports[exportedVar].type,
+                originatingPath: originatingPath,
+                hasErrorMessages: false,
+                modified: false,
+                exportedFunction: false, // FIXME: Check if type covers this. exportedVarIsFunction,
```

Comment:
Was this `FIXME` something to be fixed as part of the PR or for later?

Reply:
Fixed in 3f385e6b8.